src/index.js

```diff
@@ -0,0 +1,40 @@
+document.getElementById("up-button").onclick = function(){
+    var tempInt = parseInt(document.getElementById("temperature").innerHTML);
+    tempInt++;
+    document.getElementById("temperature").innerHTML = tempInt;
+    tempColor(tempInt);
+}
+
+document.getElementById("down-button").onclick = function(){
+    var tempInt = parseInt(document.getElementById("temperature").innerHTML);
+    tempInt--;
+    document.getElementById("temperature").innerHTML = tempInt;
+    tempColor(tempInt);
```

Comment:
Notice how similar the body of this function is to the event handler registers for increasing the temperature. Consider making a helper function to hold the parts in common. Just keep the parts that differ in these functions, and make use of the new helper for the things they do that are the same.